source/docs/3/origin_destination_matrix.rst

```diff
@@ -1,11 +1,11 @@
 Locating Nearest Facility with Origin-Destination Matrix (QGIS3)
 ================================================================
 
-In the previous tutorial, :doc:`basic_network_analysis`, we learnt how to build a network and calculate the shortest path between 2 points. We can apply that technique for many different types of network-based analysis. One such application is to compute **Origin-Destination Matrix** or **OD Matrix**. Given a set of origin points and another set of destination points, we can calculate shortest path between each origin-destination pairs and find out the travel distance/time between them. Such analysis is useful to locate the closest facility to any given point. For example, a logistics company may use this analysis to find the closest warehouse to their customers to optimize delivery routes. Here we use Distance Matrix algorithm from **QGIS Network Analysis Toolbox (QNEAT3)** plugin to find the nearest health facility to each address in the city.
+In the previous tutorial, :doc:`basic_network_analysis`, we learned how to build a network and calculate the shortest path between 2 points. We can apply that technique to many different types of network-based analysis. One such application is to compute **Origin-Destination Matrix** or **OD Matrix**. Given a set of origin points and another set of destination points, we can calculate the shortest path between each origin-destination pairs and find out the travel distance/time between them. Such analysis is useful to locate the closest facility to any given point. For example, a logistics company may use this analysis to find the closest warehouse to their customers to optimize delivery routes. Here we use the Distance Matrix algorithm from **QGIS Network Analysis Toolbox (QNEAT3)** plugin to find the nearest health facility to each address in the city.
 
 .. note::
 
-  This tutorial shows how to use your own network data to compute an origin-destination matrix. If you do not have your own network data, you can use **ORS Tools Plugin** and algorithm :menuselection:`ORS Tools --> Matrix -->  Matrix from Layers` to do the similar analysis using OpenStreetMap data. See :doc:`service_area_analysis` to learn how to use ORS Tools plugin.
+  This tutorial shows how to use your own network data to compute an origin-destination matrix. If you do not have your own network data, you can use **ORS Tools Plugin** and algorithm :menuselection:`ORS Tools --> Matrix -->  Matrix from Layers` to do a similar analysis using OpenStreetMap data. See :doc:`service_area_analysis` to learn how to use ORS Tools plugin.
 
 Overview of the task
```

Comment:
Other skills, remove 'stratified sample' nad making it 'random sample'

Reply:
Updated.